app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -81,12 +81,14 @@ fun MapScreen(
   val mapViewportState = MapConfig.createMapViewPortStateFromViewModel(mapViewModel)
   var removeViewAnnotation = remember { true }
   var cancelables = remember<Cancelable> { Cancelable({}) }
+  var listener = remember<MapIdleCallback?> { null }
   var pointAnnotationManager by remember { mutableStateOf<PointAnnotationManager?>(null) }
+  val selectedParking = parkingViewModel.selectedParking.collectAsState().value
```

Comment:
I think calling .collectAsState().value is useless, if the state is updated the selectedParking val is not updated when writing this.
you should just use the `by` syntax

Reply:
Well, technically speaking we collect a State and then I extract the `value` attribute. Collecting a state first means that it will be updated on change and then extract the `value` no ?

Reply:
I agree w/ @Pijukai here. This seems like the most sound way to get the selectedParking to update given how the updateParking() function is programmed